pkg/oc/cli/admin/migrate/migrator.go

```diff
@@ -594,12 +594,14 @@ type migrateTracker struct {
 func (t *migrateTracker) report(prefix string, info *resource.Info, err error) {
 	ns := info.Namespace
 	if len(ns) > 0 {
-		ns = "-n " + ns
+		ns = " -n " + ns
 	}
+	groupResource := info.Mapping.Resource.GroupResource()
+	groupResourceStr := (&groupResource).String()
 	if err != nil {
-		fmt.Fprintf(t.out, "E%s %-10s %s %s/%s: %v\n", timeStampNow(), prefix, ns, info.Mapping.Resource.Resource, info.Name, err)
+		fmt.Fprintf(t.out, "E%s %-10s%s %s/%s: %v\n", timeStampNow(), prefix, ns, groupResourceStr, info.Name, err)
 	} else {
-		fmt.Fprintf(t.out, "I%s %-10s %s %s/%s\n", timeStampNow(), prefix, ns, info.Mapping.Resource.Resource, info.Name)
+		fmt.Fprintf(t.out, "I%s %-10s%s %s/%s\n", timeStampNow(), prefix, ns, groupResourceStr, info.Name)
```

Comment:
why did you remove the space here?

Reply:
Should only be used if namespace is non-empty.